cmd/cloud/deploy.go

```diff
@@ -68,6 +68,22 @@ func newDeployCmd() *cobra.Command {
 	return cmd
 }
 
+func deployTests(pars, pytest, forceDeploy bool, pytestFile string) string {
+	if pytest && pytestFile == "" {
+		pytestFile = "all-tests"
+	}
+
+	if !parse && !pytest && !forceDeploy || parse && !pytest && !forceDeploy || parse && !pytest && forceDeploy {
```

Comment:
Regardless if `!parse` or `parse` pytestFile always has `"parse"`. Was wondering if dropping that check would make this condition a little easier to read?

Reply:
by default pytestFile is empty string...so if i remove that condition, `astrolocal deploy <deployment-id` would not parse dags, which is not what we want

Reply:
Makes sense. Thanks